### PR TITLE
Add version to modelDescription.xml

### LINKED ISSUE #39: Add PythonFMU version to modelDescription.xml
Version should be read from _version.py

### PR DESCRIPTION
**Pull request by markaren**
Closes #39

### CODE REVIEW

File: pythonfmu/tests/test_fmi2slave.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import pytest
 
 from pythonfmu import Fmi2Slave, Boolean, Integer, Real, String
+from pythonfmu import __version__ as VERSION
```
Comment (markaren) [inline pythonfmu/tests/test_fmi2slave.py]: lol. Silly me

**Comment by markaren** [comment]
Mhh.. the PyCharm formatter uses more indents :/ We should probably standardize on the formatiing rules.

**Comment by markaren** [comment]
It looks to be the `Continuation indent` setting is set to 8 in PyCharm, while you're formatter uses 6 as indent.

File: pythonfmu/tests/test_fmi2slave.py (hunk 1 above)
Comment (fcollonval) [reply pythonfmu/tests/test_fmi2slave.py]: 😄

**Comment by fcollonval** [comment]
This is weird, the convention in Python is to use N * 4 spaces as indent convention. Where did you see a 6 spaces indent?
What I know to be different in PyCharm compare to black is the handling of long string and maybe the line length (black uses 88).

For reference the [black rules](https://black.readthedocs.io/en/stable/the_black_code_style.html)

**Comment by markaren** [comment]
In was wrong with 6 in black. I'ts 4

**Comment by markaren** [comment]
I can set the continuation indent to 4 in PyCharm. Merging this.

**Pull request merged**